.evergreen/run-ocsp-tests.sh

```diff
@@ -14,7 +12,6 @@ source "${PROJECT_DIRECTORY}/.evergreen/init-nvm.sh"
 #     PYTHON=python
 
 # NOTE: `--opts {}` is used below to revert mocha to normal behavior (without mongodb specific plugins)
-MONGODB_UNIFIED_TOPOLOGY=${UNIFIED} \
 OCSP_TLS_SHOULD_SUCCEED=${OCSP_TLS_SHOULD_SUCCEED} \
 CA_FILE=${CA_FILE} \
 npm run check:ocsp
```

Comment:
Can we split the env vars like in the main test script here?

Reply:
not really spec compliance related, but sure